src/Combobox/ComboboxCategory.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+interface ComboboxCategoryProps {
+  children: React.ReactNode;
+  label?: string;
+}
+
+const ComboboxCategory: React.FC<ComboboxCategoryProps> = ({ children }) => (
```

Comment:
`React.FC` is less popular these days: https://github.com/facebook/create-react-app/pull/8177

```
const ComboboxCategory = ({ children }: React.PropsWithChildren<{}>) => ...
```

for the alternative.